docs/snippets/README.md

```diff
@@ -0,0 +1,8 @@
+# Snippets
```

Comment:
might be worth putting the `snippets` dir in `base_paths` so the path doesn't need to be specified (see https://facelessuser.github.io/pymdown-extensions/extensions/snippets/#specifying-snippet-locations). Also if the snippets dir lives under docs/ we should probably add it to [exclude](https://github.com/knative/docs/blob/mkdocs/mkdocs.yml#L276) so it doesn't get built.

Reply:
I will do this now thank you!!